package.json

```diff
@@ -49,14 +49,16 @@
     "run-test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "lint": "eslint --ignore-path .gitignore .",
-    "now-start": "node server.js",
-    "start-dev": "NODE_ENV=development nodemon --ignore ./src server.js",
-    "lint:fix": "eslint --ignore-path .gitignore --fix ."
+    "now-start": "npm run compile-server && node lib/index.js",
+    "start-dev": "NODE_ENV=development nodemon lib/index.js",
+    "lint:fix": "eslint --ignore-path .gitignore --fix .",
+    "compile-server": "rimraf lib/ && babel -d lib/ server/"
```

Comment:
Did you install `rimraf`? Otherwise this doesn't work...

Reply:
Yea I didn't notice I hadn't. It was working because other dependencies require it. Updated. Good catch!